plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -74,4 +77,18 @@ public IcebergConfig setUseFileSizeFromMetadata(boolean useFileSizeFromMetadata)
         this.useFileSizeFromMetadata = useFileSizeFromMetadata;
         return this;
     }
+
+    @Min(1)
+    public int getMaxPartitionsPerWriter()
+    {
+        return maxPartitionsPerWriter;
+    }
+
+    @Config("iceberg.max-partitions-per-writers")
```

Comment:
nit: `writers` -->`writer`? I think it'd be okay to diverge from hive connector's config for in interest of better naming. @electrum any preference?

Reply:
updated to `iceberg.max-partitions-per-writer`